Travel Scheduler/View Controllers/MapViewController.m

```diff
@@ -0,0 +1,239 @@
+//
+//  MapViewController.m
+//  Travel Scheduler
+//
+//  Created by frankboamps on 7/31/19.
+//  Copyright © 2019 aliu18. All rights reserved.
+//
+
+#import "MapViewController.h"
+#import "Place.h"
+#import <CoreLocation/CoreLocation.h>
+#import "TravelSchedulerHelper.h"
+
+@interface MapViewController () <CLLocationManagerDelegate>
+
+- (NSArray*) calculateRoutesFrom:(CLLocationCoordinate2D) from to: (CLLocationCoordinate2D) to;
```

Comment:
You don't need that many spaces in the signature. Should be:
- (NSArray*)calculateRoutesFrom:(CLLocationCoordinate2D)from to:(CLLocationCoordinate2D)to;

^Only one space after the dash and one space between parameters.